integration-cli/docker_cli_exec_test.go

```diff
@@ -25,9 +25,7 @@ func (s *DockerSuite) TestExec(c *check.C) {
 
 	out, _ := dockerCmd(c, "exec", "testing", "cat", "/tmp/file")
 	out = strings.Trim(out, "\r\n")
-	if out != "test" {
-		c.Errorf("container exec should've printed test but printed %q", out)
-	}
+	c.Assert(out, checker.Equals, "test")
```

Comment:
@vdemeester thanks, but in the PR #17004 @cpuguy83 has suggested me to remove the comment, he said that the error message on failure will give the needed information in these cases.